formbricks/formbricks
# chore: Redesign metadata

## What does this PR do?


Fixes 1271 (https://linear.app/formbricks/issue/FOR-1271/redesign-metadata)

- Redesigned Metadata 
- Also adds a functionality to remove display associated with a response when a response is deleted

<img width="666" alt="Screenshot 2023-09-27 at 2 09 28 PM" src="https://github.com/formbricks/formbricks/assets/67850763/bcaf0c5c-8bff-44d8-abf1-b4cce8ae4dce">






## Type of change




## How should this be tested?



Add some responses to a survey
Open summary page and delete those responses
You'll notice that along with response, display is also removed

## Checklist
